doc/api-docs/conf.py

```diff
@@ -12,18 +12,25 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys
+import importlib
+import itertools
 import os
-import imp
-import ast
-import pprint
+import pkgutil
+import subprocess
+import sys
+import warnings
+
+from sphinx.ext import apidoc
+from unittest.mock import MagicMock
+
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 DOCS_SOURCE_DIR = os.path.dirname(__file__)
-sys.path.insert(0, os.path.join(DOCS_SOURCE_DIR, '..', '..', 'src'))
-pprint.pprint(sys.path)
+INVEST_ROOT_DIR = os.path.join(DOCS_SOURCE_DIR, '..', '..')
+INVEST_SOURCE_DIR = os.path.join(INVEST_ROOT_DIR, 'src')
+sys.path.insert(0, INVEST_SOURCE_DIR)
```

Comment:
I forget why, but I was doing this so that I could build the API docs from the source directory rather than the build directory. Changed it to build docs directly from the `build/lib` directory in 02b6ef7